rootfs/templates/kops/default.yaml

```diff
@@ -31,10 +31,26 @@ spec:
   api:
     loadBalancer:
       type: Public
-  {{- if bool (getenv "KOPS_AUTHORIZATION_RBAC_ENABLED" "false") }}
+  hooks:
+  {{- if bool (getenv "KOPS_AWS_IAM_AUTHENTICATOR_ENABLED" "false") }}
+    - name: kops-hook-authenticator-config.service
+      before:
+        - kubelet.service
+      roles: [Master]
+      manifest: |
```

Comment:
Shouldn't it be?
```
manifest: |-
```

Reply:
i picked it from official documentation.
`|-` just removes \n from the end